vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridViewSelectionIT.java

```diff
@@ -43,146 +42,6 @@ public void init() {
         open();
     }
 
-    @Test
-    public void gridAsSingleSelect() {
-        GridElement grid = $(GridElement.class).id("single-selection");
-        scrollToElement(grid);
-
-        WebElement toggleButton = findElement(By.id("single-selection-toggle"));
-        WebElement messageDiv = findElement(By.id("single-selection-message"));
-
-        clickElementWithJs(toggleButton);
-        Assert.assertEquals(
-                getSelectionMessage(null, LegacyTestView.items.get(0), false),
-                messageDiv.getText());
-        Assert.assertTrue("Person 1 was not marked as selected",
-                isRowSelected(grid, 0));
-        clickElementWithJs(toggleButton);
-        Assert.assertEquals(
-                getSelectionMessage(LegacyTestView.items.get(0), null, false),
-                messageDiv.getText());
-        Assert.assertFalse("Person 1 was marked as selected",
-                isRowSelected(grid, 0));
-
-        // should be the cell in the first column's second row
-        clickElementWithJs(getCell(grid, "Person 2"));
-        Assert.assertTrue("Person 2 was not marked as selected",
-                isRowSelected(grid, 1));
-        Assert.assertEquals(
-                getSelectionMessage(null, LegacyTestView.items.get(1), true),
-                messageDiv.getText());
-        clickElementWithJs(getCell(grid, "Person 2"));
-        Assert.assertFalse("Person 2 was marked as selected",
-                isRowSelected(grid, 1));
-
-        clickElementWithJs(getCell(grid, "Person 2"));
-        clickElementWithJs(toggleButton);
-        Assert.assertTrue("Person 1 was not marked as selected",
-                isRowSelected(grid, 0));
-        Assert.assertFalse("Person 2 was marked as selected",
-                isRowSelected(grid, 1));
-        Assert.assertEquals(
-                getSelectionMessage(LegacyTestView.items.get(1),
-                        LegacyTestView.items.get(0), false),
-                messageDiv.getText());
-        clickElementWithJs(toggleButton);
-        Assert.assertFalse("Person 1 was marked as selected",
-                isRowSelected(grid, 0));
-
-        // scroll to bottom
-        scroll(grid, 1000);
-        waitUntilCellHasText(grid, "Person 499");
-        // select item that is not in cache
-        clickElementWithJs(toggleButton);
-        Assert.assertEquals(
-                getSelectionMessage(null, LegacyTestView.items.get(0), false),
-                messageDiv.getText());
-        // scroll back up
-        scroll(grid, 0);
-        Assert.assertTrue("Person 1 was not marked as selected",
-                isRowSelected(grid, 0));
-
-        Assert.assertFalse(
-                getLogEntries(Level.SEVERE).stream().findAny().isPresent());
-    }
-
-    @Test
-    public void gridAsSingleSelectTestBenchAPI() {
```

Comment:
This is the same test as above, except that it uses a different test APIs? First time I see this, and this is definitely not a common practice. I think it's OK to remove this.